planner-api/src/main/java/pl/edu/agh/kuce/planner/goal/GoalController.java

```diff
@@ -33,6 +33,7 @@ public GoalData create(@Valid @RequestBody final GoalInputData data, @Current fi
     }
 
     @GetMapping("/goals")
+    @ResponseStatus(HttpStatus.CREATED)
```

Comment:
Isn't it a mistake? HttpStatus.CREATED should be returned by a _create_ method a few lines up :)